windows/hax_entry.h

```diff
@@ -163,4 +163,8 @@ extern PDRIVER_OBJECT HaxDriverObject;
 #define HAX_VM_IOCTL_NOTIFY_QEMU_VERSION \
         CTL_CODE(HAX_DEVICE_TYPE, 0x910, METHOD_BUFFERED, FILE_ANY_ACCESS)
 
+/* API version 5.0 */
```

Comment:
The HAXM API is indeed versioned, and the current version is 4 (`HAX_CUR_VERSION`). When we update the API, we could bump the API version, but we could also add a "capability flag" (see `API.md` for details). I'd actually prefer the latter approach, since we're only adding one ioctl. As an example, please take a look at commit 43ff9f0.